backend/dataall/core/environment/cdk/environment_stack.py

```diff
@@ -697,7 +697,7 @@ def create_integration_tests_role(self):
                 ],
                 effect=iam.Effect.ALLOW,
                 resources=[
-                    f'arn:aws:iam::{self.account}:role/dataall-test-*',
+                    f'arn:aws:iam::{self.account}:role/dataall-test*',
```

Comment:
This was made, so the int test role could ListPolicies for role `dataall-testgroup-*`